integration/dagster/openlineage/dagster/sensor.py

```diff
@@ -28,13 +28,15 @@ def openlineage_sensor(
         description: Optional[str] = "OpenLineage sensor tails Dagster event logs, "
                                      "converts Dagster events into OpenLineage events, "
                                      "and emits them to an OpenLineage backend.",
+        concerned_event_types: Optional[Union[DagsterEventType, Set[DagsterEventType]]] = PIPELINE_EVENTS | STEP_EVENTS, # noqa: E501
```

Comment:
Why this is optional?
In `get_event_log_records` it's just
`event_type: Union[DagsterEventType, Set[DagsterEventType]]`

Reply:
@JDarDagran I am sorry to make you confused. I forgot to remove that `Optional`. Originally `concerned_event_types` was designed as `Optional`, the default value (`PIPELINE_EVENTS | STEP_EVENTS`) was given when calling `get_event_log_records` if that argument is `None`. To get users to know the default value without digging into the source code, I decided to set up the default value in the factory function parameter but forgot to remove `Optional`.

Here's the code snippets of original implementation.

```python
def openlineage_sensor(
        name: Optional[str] = "openlineage_sensor",
        description: Optional[str] = "OpenLineage sensor tails Dagster event logs, "
                                     "converts Dagster events into OpenLineage events, "
                                     "and emits them to an OpenLineage backend.",
        concerned_event_types: Optional[Union[DagsterEventType, Set[DagsterEventType]]] = None,
        minimum_interval_seconds: Optional[int] = DEFAULT_SENSOR_DAEMON_INTERVAL,
        record_filter_limit: Optional[int] = 30,
        after_storage_id: Optional[int] = 0
) -> SensorDefinition:
    @sensor(    # type: ignore
        name=name,
        minimum_interval_seconds=minimum_interval_seconds,
        description=description,
    )
   def _openlineage_sensor(context: SensorEvaluationContext):
        ...
        ...
        event_log_records = get_event_log_records(
            context.instance, concerned_event_types or PIPELINE_EVENTS | STEP_EVENTS, last_storage_id, record_filter_limit
        )
        ...
        ...
```

Could I correct this in my local branch and push back to the remote branch??

Reply:
Yes, please 👍